.github/workflows/ci.yml

```diff
@@ -3,16 +3,16 @@ name: CI
 on:
   push:
     paths-ignore:
-      - '*.md'
-      - 'LICENSE'
-      - 'index.d.ts'
+      - "*.md"
```

Comment:
I'm not sure we should change the quote character especially since the Github docs use single quote: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-ignoring-paths